composer.json

```diff
@@ -16,7 +16,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.1|^8.0",
+        "php": "^7.4|^8.0",
```

Comment:
I'd rather have a BC way contributed to 2.9.x and then whatever requires 7.4 added to 3.0.x. The less divergence between branches, the easier the merge up. And we definitely want to be able to merge up, unlike, you know, previously :grimacing: